src/nn.h

```diff
@@ -352,6 +364,7 @@ NN_EXPORT int nn_setsockopt (int s, int level, int option, const void *optval,
     size_t optvallen);
 NN_EXPORT int nn_getsockopt (int s, int level, int option, void *optval,
     size_t *optvallen);
+NN_EXPORT int nn_setopt (int option, const void *optval, size_t optvallen);
```

Comment:
One *could* want to set the strategy to NN_FORK_RESET, then reset it to NN_FORK_NONE in the child after fork, notably if the child itself forks then execs as the runtime cost of resetting the library could be non-negligible.